sdk/formrecognizer/Azure.AI.FormRecognizer/tests/SessionRecords/FormTrainingClientLiveTests/CopyModel.json

```diff
@@ -1,40 +1,43 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://mariari-fr-westcentralus.cognitiveservices.azure.com/formrecognizer/v2.0-preview/custom/models",
+      "RequestUri": "https://camaiaor-formrec-westcentralus.cognitiveservices.azure.com/formrecognizer/v2.0-preview/custom/models",
       "RequestMethod": "POST",
       "RequestHeaders": {
-        "Content-Length": "236",
+        "Content-Length": "42",
         "Content-Type": "application/json",
         "Ocp-Apim-Subscription-Key": "Sanitized",
-        "Request-Id": "|ff6e186c-42bf0ca2f418ce26.",
+        "Request-Id": "|d3b56c21-4b8fa537f4924b3d.",
         "User-Agent": [
-          "azsdk-net-AI.FormRecognizer/1.0.0-dev.20200613.1",
+          "azsdk-net-AI.FormRecognizer/1.0.0-dev.20200617.1",
           "(.NET Core 4.6.28516.03; Microsoft Windows 10.0.18363 )"
         ],
         "x-ms-client-request-id": "ff3644f4118ac1883154e5a60f531695",
         "x-ms-return-client-request-id": "true"
       },
-      "RequestBody": null,
+      "RequestBody": {
+        "source": "Sanitized",
```

Comment:
It seems that Copy also has a `source` property that's being sanitized. Do you know if it's safe and possible to expose it without exposing the SAS URL in training recordings?

Reply:
this source is the training documents URL, so we want to sanitize it.